homeassistant/components/motionmount/__init__.py

```diff
@@ -0,0 +1,47 @@
+"""The Vogel's MotionMount integration."""
+from __future__ import annotations
+
+import motionmount
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST, CONF_PORT, Platform
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+from .coordinator import MotionMountCoordinator
+
+PLATFORMS: list[Platform] = [
+    Platform.NUMBER,
+]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Vogel's MotionMount from a config entry."""
+
+    hass.data.setdefault(DOMAIN, {})
+
+    # Create API instance
+    mm = motionmount.MotionMount(entry.data[CONF_HOST], entry.data[CONF_PORT])
+    coordinator = MotionMountCoordinator(hass, mm)
+    mm.add_listener(coordinator.motionmount_callback)
```

Comment:
```suggestion
    mm.add_listener(hass.add_job(coordinator.async_request_refresh))
```
Just asked someone else, can you give this a shot?

Reply:
That gives me "'NoneType' object is not callable".
It works if I move this new code into the `motionmount_callback()`, but it makes the system rather slow in updates as the update request is delayed. This might be a feature, but it is nice to see the sliders respond almost instantly to updates.

Reply:
I think there is also an `async_refresh` `request` waits a few seconds to group any other requests

Reply:
Indeed! I've updated the code. It still uses the `motionmount_callback()` to avoid the 'object not callable' error.